### PR TITLE
Add Fritzing

### PR DESCRIPTION
**Pull request by gasinvein**
Fritzing is a simple yet full-featured EDA.
Thanks openSUSE for [their patches](https://build.opensuse.org/package/show/Education/fritzing).

### CODE REVIEW

File: org.fritzing.Fritzing.json [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+        "/lib/cmake",
+        "/share/man"
+    ],
+    "modules": [
+        {
+            "name": "boost",
+            "buildsystem": "simple",
+            "build-commands": [
+                "./bootstrap.sh",
+                "./b2 install variant=release link=shared runtime-link=shared --prefix=/app -j $FLATPAK_BUILDER_N_JOBS"
```
Comment (barthalion) [inline org.fritzing.Fritzing.json]: You can limit which boost libraries are being built to shorten the build time (and make the app smaller) with `--with-libraries=`.

**Comment by barthalion** [comment]
Have you contacted upstream developers if they would like to help with maintenance/get RW access to the repo?

bot, build org.fritzing.Fritzing

**Comment by gasinvein** [comment]
No. Should I?

**Comment by nedrichards** [comment]
bot, build org.fritzing.Fritzing

**Comment by barthalion** [comment]
Nothing in that issue concerns Flatpak or I'm missing something? The general idea is to at least give upstream a heads-up about adding their project to Flathub.

**Comment by gasinvein** [comment]
That issue is about packaging Fritzing in general. Latest version (0.9.3b), without heavy patching, is unfriendly to automated builds.
Anyway, I'll notify the developers about ther app being submitted to Flathub.

**Comment by gasinvein** [comment]
On Fritzing issues, there already was a request of flatpak. Posted there. But the developers seem inactive currently.

**Comment by barthalion** [comment]
That's ok, it's not a requirement to have author's blessing currently,

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/org.fritzing.Fritzing

Thanks!

**Pull request closed without merging**